config/software/ncurses.rb

```diff
@@ -23,6 +23,7 @@
 skip_transitive_dependency_licensing true
 
 dependency "config_guess"
+dependency "libtool" if aix?
```

Comment:
this seems wrong, because libtool is a build time dep, and it should only go into omnibus-toolchain, by putting it here, anything shipping software that pulls in ncurses will pull in libtool and then ship it.

if this is necessary to build omnibus-toolchain then the ordering should probably go into the project definition.